src/components/HomeCard/home-card.styles.jsx

```diff
@@ -0,0 +1,26 @@
+import styled from 'styled-components';
+import Container from '../../utils/components/Container';
+
+
+export const Circle = styled.div`
```

Comment:
I added the .jsx extension because I used two extensions. (style and jsx)

When you create a file _home-card.component_ or _home-card.styles_ and then you import them using ``import HomeCard from "./home-card.component"`` by default the compiler will be looking for .jsx extensions instead of .js and will throw an error if you use ``home-card.component.js``

However, if you want to use the .js extension, a quick fix would be changing the import statement to ``import HomeCard from "./home-card.component.js"

This is just a personal preference. I like separating my code into .component and .styles so I know exactly what each file is for.

However, I changed the file extensions to .js, as you requested.